replays/battle.template.php

```diff
@@ -174,13 +174,9 @@ function userid($username) {
 
 		<input type="hidden" name="replayid" value="<?php echo htmlspecialchars($replay['id']); ?>" />
 		<!--
-
 You can get this log directly at https://replay.pokemonshowdown.com/<?php echo $replay['id']; ?>.log
-
 Or with metadata at https://replay.pokemonshowdown.com/<?php echo $replay['id']; ?>.json
-
 Most PS pages you'd want to scrape will have a .json version!
-
```

Comment:
Is this necessary? :( I like the spacing to make it easier to see.

Reply:
No, it's not necessary - I'm not actually sure why I changed that.